### ISSUE TITLE
Compatibility question

### ISSUE THREAD

**Issue by hmlendea** [Gr770/CK3-Community-Title-Project#64]
Hello,

I'm not very experienced modding CK3 titles yet, so I thought I should ask to make sure.

Do you happen to know how compatible are localisation mods with each other?

I was thinking of updating my mod (https://github.com/hmlendea/more-cultural-names) which already has title localisations for CK2, to also have them for CK3.

However, I am aware that there is a considerable amount of people who use both of our mods, and I want to make sure they can continue to do so (our mods do go really very well together).

I'm also going to try to make pull requests to this repository to include them here as well (in fact, I've already made a PR for `vlach`: #63)

My worry is for when we both, inevitably, localise the same titles. As far as I can tell, the `priority` property should take care of "conflicts", but is that enough, is there something I am missing?

**Comment by Charanraja123** [comment]
Hello. 
I think we are all kinda of starting out in CK3, so none of us are experienced in any way. 
Compatibility of Titles across mod is frankly a little tricky and confusing. Let me explain.
There are basically two factors which affect it.
1) Mod loading order.
2) Priority number.
 
From what I could learn, for flavorisation, this is how the loading up works. The game merges all the files together. If two mods have the same keys (Eg: baron_feudal_male in both mods), then the one with the higher in the mod loading overrides the lower one. 

Next, after merging, it checks as usual and decides based on priority number. So in essence, we have to make sure our priority numbers work in tandem with each other. If a character qualifies for two different definition, and I have a priority number greater than you, Then yours will not show up (and vice versa). I've made an explanation to what happens when priority numbers are equal below.

Point to note, when merging the mod file usually appends to the bottom of the vanilla file, then the mod higher than that in the loading order is appended next. Now this means the mod with the highest loading order ends up at the last, which is good because if a character qualifies for different definitions with same priority, it picks the last one (According to Paradox, they said this in that file). 
Now the above point is what I call expected behavior from what I see. Whether Paradox implemented this correctly is something I have not tested 

TLDR: It requires a lot of working together to pull off.

All said, I would definitely want us to work together. I think there is no point in splitting up resources/having competition in the modding community unless you find the other person/team intolerable to work with (I really don't think that is the case for us ). I hope you join us, But i leave the decision in your hands.

Regarding the Vlach pull request, @Viridianus is the one who reviews that. He'll be responding to that soon, Unless he is somewhere deep in the sheets clearing up things xP

**Comment by hmlendea** [comment]
Thanks for explaining.
Seems rather complicated and indeed a waste of resources. I was hoping it would be something more simple, but apparently not quite.

It's gonna be simpler then if I don't cover titles in the CK3 version of my mod and instead direct users to use this mod instead.
Which is actually quite good to be honest, as maybe some would only want the places localised, or others just the titles. It gives more control to the user this way.

I will try to follow up with a few more pull requests later, but I don't know when.

**Comment by Gr770** [comment]
I'm open to it, you can use the landed titles sheet and direct people who would like to contribute to the sheet as well.

**Comment by Viridianus** [comment]
This doesn't seem to have much more to discuss so I'm closing it.
